src/bun.js/webcore/response.zig

```diff
@@ -884,6 +886,12 @@ pub const Fetch = struct {
                     if (options.get(globalThis, "verbose")) |verb| {
                         verbose = verb.toBoolean();
                     }
+                    if (options.get(globalThis, "signal")) |signal_arg| {
+                        if (signal_arg.as(JSC.AbortSignal)) |signal_| {
+                            _ = signal_.ref();
```

Comment:
We shouldn't need to reference count the signal here since we keep the JSValue alive

Reply:
Yeah, I was trying to keep them alive, in but I failed, I will remove this